azure-pipelines.yml

```diff
@@ -100,21 +100,31 @@ jobs:
     - script: |
         cmake -DPACKAGING_SYSTEM=DEB $(Build.SourcesDirectory)
         cmake --build . --target package -j 3
+        cp *.deb $(Build.ArtifactStagingDirectory)
```

Comment:
After building packages I run:

```
mv osquery_$VERSION-1.linux_$ARCH.deb osquery_$VERSION_1.linux.$ARCH.deb
```

Reply:
> Just a heads up about the differences between how we store files in S3 and what our CPack output creates.

Curiosity, I see that is how we are still doing things, but is there a reason why we are not following the standard naming convention for each packaging format?
The PR https://github.com/osquery/osquery/pull/6468 intention was to adhere to the standard naming given by the underlying tools.
If we don't want to follow the standard naming convention, I think the best course is to revert that PR then, instead of having to do an additional move step.